### ISSUE TITLE
Guild ID of length 19 fails assertion

### ISSUE THREAD

**Issue by PiggyPlex** [ghostdevv/jellycommands#143]
### Describe the bug

Errors if a server ID of length 19 is used in dev.guilds in the index.js file.
This is because it is parsed against a Snowflake filter that accepts only length 18.
```js
import 'dotenv/config';
import { JellyCommands } from 'jellycommands';
import { IntentsBitField } from 'discord.js';

const client = new JellyCommands({
    // https://jellycommands.dev/guide/commands/loading
    commands: 'src/commands',

    // https://jellycommands.dev/guide/events/loading
    events: 'src/events',

    clientOptions: {
        intents: [IntentsBitField.Flags.Guilds],
    },

    dev: {
        // In testing we should enable this, it will make all our commands register in our testing guild
        // https://jellycommands.dev/guide/commands/dev#global-dev-mode
        global: true,

        // Put your testing guild id here
        // https://jellycommands.dev/guide/commands/dev#setup
        guilds: ['1012921113422078022'],
    },
});

// Automatically reads the DISCORD_TOKEN environment variable
client.login();
```

### Reproduction

1. Create a new Discord guild.
2. Enter it into index.js as a development testing guild.
3. pnpm dev

### Logs

```shell
[1] "dev.guilds[0]" length must be 18 characters long
```

### System Info

```shell
System:
    OS: Windows 10 10.0.19043
    CPU: (8) x64 Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz
    Memory: 948.69 MB / 7.88 GB
  Binaries:
    Node: 16.17.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.15.0 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    discord.js: ^14.2.0 => 14.3.0
    jellycommands: 1.0.0-next.32 => 1.0.0-next.32
```

**Comment by ghostdevv** [comment]
Great catch thanks!

**Comment by ghostdevv** [comment]
Fixed in 129e61d89d892be0602378a98d5be1f6d4dbb333
Released in `1.0.0-next.37`

Thanks for reporting!
